src/contentsharecontroller/ContentShareMediaStreamBroker.ts

```diff
@@ -56,24 +57,32 @@ export default class ContentShareMediaStreamBroker implements MediaStreamBroker
     throw new Error('unsupported');
   }
 
-  async acquireScreenCaptureDisplayInputStream(sourceId?: string): Promise<MediaStream> {
-    return this.acquireDisplayInputStream(this.screenCaptureDisplayMediaConstraints(sourceId));
+  async acquireScreenCaptureDisplayInputStream(
+    sourceId?: string,
+    frameRate?: number
+  ): Promise<MediaStream> {
+    return this.acquireDisplayInputStream(
+      this.screenCaptureDisplayMediaConstraints(sourceId, frameRate)
+    );
   }
 
-  private screenCaptureDisplayMediaConstraints(sourceId?: string): MediaStreamConstraints {
+  private screenCaptureDisplayMediaConstraints(
+    sourceId?: string,
+    frameRate?: number
+  ): MediaStreamConstraints {
     return {
       audio: false,
       video: {
         ...(!sourceId && {
           frameRate: {
-            max: 3,
+            max: frameRate ? frameRate : ContentShareMediaStreamBroker.defaultFrameRate,
```

Comment:
Don't need to update in this PR but later you can just use ```||```.
``frameRate || ContentShareMediaStreamBroker.defaultFrameRate``